python/pyspark/sql/session.py

```diff
@@ -221,6 +221,9 @@ def __init__(self, sparkContext, jsparkSession=None):
                 or SparkSession._instantiatedSession._sc._jsc is None:
             SparkSession._instantiatedSession = self
 
+    def _repr_html_(self):
+        return self.sparkContext._repr_html_()
```

Comment:
> It might also make sense to return a different URL link (e.g. to the SQL page rather than the default page which takes people to the Jobs section) but this is minor and likely less useful than the other things.

I don't know enough internals here to tell what URL(s) you'd want here. What properties or calls should I make for the SQL page?